tests/components/flo/test_config_flow.py

```diff
@@ -14,26 +17,36 @@ async def test_form(hass, aioclient_mock_fixture):
     assert result["type"] == "form"
     assert result["errors"] == {}
 
-    with patch(
-        "homeassistant.components.flo.async_setup", return_value=True
-    ) as mock_setup, patch(
-        "homeassistant.components.flo.async_setup_entry", return_value=True
```

Comment:
We normally patch the integration `async_setup_entry` function on entry create result to avoid setting up the integration.

Reply:
replaced